packages/hash/api/src/graphql/resolvers/linkedAggregation/deleteLinkedAggregation.ts

```diff
@@ -13,7 +13,7 @@ export const deleteLinkedAggregation: Resolver<
   MutationDeleteLinkedAggregationArgs
 > = async (
   _,
-  { sourceAccountId, sourceEntityId, path },
+  { sourceAccountId, sourceEntityId, aggregationId },
```

Comment:
Do we still need to use the `sourceEntityId` here, and in other mutations? Previously we needed to because we needed the source to identify the aggregation, but it feels unnecessary now.

I suggest we remove the need to pass `sourceEntityId` in the queries/mutations and the fetching of the source entity in the resolvers.

Update: having read on I see we use `sourceEntityId` and check for the entity in `deleteLinks` also. Maybe we should discuss as a separate follow up, as it feels unnecessary.

Reply:
My reasoning for this was poor. I originally thought it may be good because then it will allow the source entity to be fetched concurrently to the aggregation (so that permissioning can be checked on the source entity).

But I realise that this isn't a good reason to make the GQL API harder to use, and also that we will likely end up implementing permissioning checks at the graph API level anyway (rather than the workspace GQL API). I have addressed this in https://github.com/hashintel/hash/pull/497/commits/413855ddf7ea2991b42e473c18f55e0fa9df2318

Reply:
Makes sense, thank you. Could you also update the image and video blocks to remove where they're passing `sourceEntityId` to `deleteLinks`? 

I've had a look and the only other place using `deleteLinks` is the entity editor, and that's already not passing a `sourceEntityId`

Reply:
Thanks for spotting, have done so in https://github.com/hashintel/hash/pull/497/commits/e3854b8b01f3ff9c875c5b8f91c97868ca4f7592